src/mds/Mutation.cc

```diff
@@ -173,6 +173,10 @@ void MutationImpl::cleanup()
   drop_pins();
 }
 
+void MutationImpl::_dump_op_descriptor_unlocked(ostream& stream) const
+{
+  stream << "Mutation";
```

Comment:
They don't actually get tracked since with don't initialize them with an OpTracker*, so it doesn't really matter.  We just have to implement the method.